package.json

```diff
@@ -19,13 +19,13 @@
   "author": "Consumer Financial Protection Bureau",
   "license": "CC0-1.0",
   "dependencies": {
-    "csv-parse": "4.13.1",
+    "csv-parse": "^4.14.2",
```

Comment:
Can we install the latest fully pinned version of csv-parse (4.15.0)?

Reply:
Good eye, thank you.